src/app/pages/viewing/responses-list/responses-list.component.css

```diff
@@ -81,26 +86,24 @@
 
 /* Conditionally-activated class */
 .nav-mobile-open {
-  left: 0;
-  width: 100%;
+  right: 0;
 }
 
 /* Button component */
 .hamburger-mobile {
```

Comment:
There is a shadow here next to the arrow that can be seen when the hamburger menu opens (see a bit below and to the right of the X in the picture)
![image](https://user-images.githubusercontent.com/24356367/88462151-2f680e00-ce77-11ea-85fc-04d60dfd281d.png)